08-transport.md

```diff
@@ -513,116 +405,92 @@ The structure of packets on the wire resembles the following:
-
   * Send `lc || c` over the network buffer.
 
 
 ### Decrypting Messages
 
-
 In order to decrypt the _next_ message in the network stream, the following is
 done:
 
-
-  * Read _exactly_ `18-bytes` from the network buffer.
-
-
+  * Read _exactly_ 18 bytes from the network buffer.
   * Let the encrypted length prefix be known as `lc`
-
-
-  * Decrypt `lc` using `ChaCha20-Poly1305`, `rn`, and `rk` to obtain size of
+  * Decrypt `lc` using `ChaCha20-Poly1305`, `rn`, and `rk`, to obtain the size of
     the encrypted packet `l`.
     * A zero-length byte slice is to be passed as the AD (associated data).
     * The nonce `rn` MUST be incremented after this step.
-
-
   * Read _exactly_ `l+16` bytes from the network buffer, let the bytes be known as
     `c`.
-
-
-  * Decrypt `c` using `ChaCha20-Poly1305`, `rn`, and `rk` to obtain decrypted
+  * Decrypt `c` using `ChaCha20-Poly1305`, `rn`, and `rk`, to obtain decrypted
     plaintext packet `p`.
-
     * The nonce `rn` MUST be incremented after this step.
 
 
 ## Lightning Message Key Rotation
 
-
-Changing keys regularly and forgetting the previous key is useful for
-preventing decryption of old messages in the case of later key leakage (ie.
+Changing keys regularly and forgetting previous keys is useful to
+prevent the decryption of old messages in the case of later key leakage (i.e.
 backwards secrecy).
 
-
-Key rotation is performed for _each_ key (`sk` and `rk`) _individually _. A key
-is to be rotated after a party sends or decrypts `1000` messages with it.
+Key rotation is performed for _each_ key (`sk` and `rk`) _individually_. A key
+is to be rotated after a party sends or decrypts 1000 messages with it.
 This can be properly accounted for by rotating the key once the nonce dedicated
-to it exceeds `1000`.
-
+to it exceeds 1000.
 
 Key rotation for a key `k` is performed according to the following:
 
-
-  * Let `ck` be the chaining key obtained at the end of `Act Three`.
+  * Let `ck` be the chaining key obtained at the end of Act Three.
   * `ck', k' = HKDF(ck, k)`
   * Reset the nonce for the key to `n = 0`.
   * `k = k'`
   * `ck = ck'`
   
-# Security Considerations #
-
+# Security Considerations
 
 It is strongly recommended that existing, commonly-used, validated
 libraries be used for encryption and decryption, to avoid the many
 implementation pitfalls possible.
 
 # Appendix A: Transport Test Vectors
 
-To make a repeatable handshake, we specify what `generateKey()` will
-return (ie. the value for `e.priv`) for each side.  Note that this
-is a violation of the spec, which requires randomness here.
+To make a repeatable test handshake, the following specifies what `generateKey()` will
+return (i.e. the value for `e.priv`) for each side. Note that this
+is a violation of the spec, which requires randomness.
 
 ## Initiator Tests
 
 The initiator should produce the given output when fed this input.
 The comments reflect internal state for debugging.
-
+```
```

Comment:
Technically indentation serves the same purpose as triple-quote in markdown, but there's nothing wrong with both AFAICT.